GrandCentralBoard/Utilities/GoogleAPIDataProvider.swift

```diff
@@ -11,10 +11,15 @@ import Operations
 
 enum APIDataError : ErrorType {
     case AuthorizationError
+    case ModelDecodeError(ErrorType)
     case UnderlyingError(NSError)
 }
 
-final class GoogleAPIDataProvider {
+protocol APIDataProvider {
```

Comment:
`APIDataFetching` / `APIDataProviding` - describe ability. 

`APIDataProvider` is better name for a class.